jetty-server/src/main/java/org/eclipse/jetty/server/ForwardedRequestCustomizer.java

```diff
@@ -420,6 +432,12 @@ else if (forwarded._server != null)
         }
     }
 
+    protected void onError(HttpField field, Throwable t)
+    {
+        LOG.warn("Exception while processing {}", field, t);
+        throw new BadMessageException("Bad header value for " + field.getName());
```

Comment:
add the Throwable t as the cause of this exception

Reply:
Ok sure, was just being careful of giving away too much information in the response like simone said.